ibm/service/vpc/data_source_ibm_is_flow_log.go

```diff
@@ -237,67 +243,76 @@ func dataSourceIBMIsFlowLogRead(context context.Context, d *schema.ResourceData,
 
 		getFlowLogCollectorOptions.SetID(d.Get("identifier").(string))
 
-		flowlogCollector, response, err := sess.GetFlowLogCollectorWithContext(context, getFlowLogCollectorOptions)
+		flowlogCollector, response, err := vpcClient.GetFlowLogCollectorWithContext(context, getFlowLogCollectorOptions)
 		if err != nil {
-			log.Printf("[DEBUG] GetFlowLogCollectorWithContext failed %s\n%s", err, response)
-			return diag.FromErr(fmt.Errorf("GetFlowLogCollectorWithContext failed %s\n%s", err, response))
+			tfErr := flex.TerraformErrorf(err, fmt.Sprintf("GetFlowLogCollectorWithContext failed: %s\n%s", err, response), "ibm_is_flow_log", "read")
+			log.Printf("[DEBUG]\n%v", tfErr.GetDiag())
+			return tfErr.GetDiag()
 		}
 		flowLogCollector = flowlogCollector
 	}
 
 	d.SetId(*flowLogCollector.ID)
 	if err = d.Set("active", flowLogCollector.Active); err != nil {
-		return diag.FromErr(fmt.Errorf("[ERROR] Error setting active: %s", err))
+		err = fmt.Errorf("Error setting active: %s", err)
+		return flex.DiscriminatedTerraformErrorf(err, err.Error(), "ibm_is_flow_log", "read", "set-active").GetDiag()
```

Comment:
is this generated code?  different from rest of the PRs 

`	if err = d.Set("is_default", addressPrefix.IsDefault); err != nil {
		tfErr := flex.TerraformErrorf(err, fmt.Sprintf("Error setting is_default"), "ibm_is_vpc_address_prefix", "read")
		log.Printf("[DEBUG] %s", tfErr.GetDebugMessage())
		return tfErr.GetDiag()
	}`